src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs

```diff
@@ -0,0 +1,1944 @@
+                }
+            }
+
+
+            remove
+            {
+                ErrorCode error = VcMgrUnsetErrorCb();
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Remove ErrorOccurred Failed with error " + error);
+                }
+
+                _errorOccurred -= value;
+                _errorDelegate = null;
+            }
+        }
+
+        /// <summary>
+        /// Event to be called when conversation requests.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/voicecontrol.manager</privilege>
+        /// <privlevel>public</privlevel>
+        /// <feature>http://tizen.org/feature/speech.control</feature>
+        /// <feature>http://tizen.org/feature/microphone</feature>
+        /// <pre>
+        /// The State should be Initialized
+        /// </pre>
+        public static event EventHandler<ConversationRequestedEventArgs> ConversationRequested
+        {
+            add
+            {
+                _conversationRequestDelegate = (int pid, IntPtr dispText, IntPtr uttText, bool continuous, IntPtr userData) =>
+                {
+                    string dispTextString = Marshal.PtrToStringAnsi(dispText);
+                    string uttTextString = Marshal.PtrToStringAnsi(uttText);
+                    ConversationRequestedEventArgs args = new ConversationRequestedEventArgs(ApplicationManager.GetAppId(pid), dispTextString, uttTextString, continuous);
+                    _conversationRequested?.Invoke(null, args);
+                };
+                ErrorCode error = VcMgrSetDialogRequestCb(_conversationRequestDelegate, IntPtr.Zero);
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Add ConversationRequested Failed with error " + error);
+                }
+                else
+                {
+                    _conversationRequested += value;
+                }
+            }
+
+            remove
+            {
+                ErrorCode error = VcMgrUnsetDialogRequestCb();
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Remove ConversationRequested Failed with error " + error);
+                }
+
+                _conversationRequested -= value;
+                _conversationRequestDelegate = null;
+            }
+        }
+
+        /// <summary>
+        /// Enable command type as candidate command.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/voicecontrol.manager</privilege>
+        /// <privlevel>public</privlevel>
+        /// <feature>http://tizen.org/feature/speech.control</feature>
+        /// <feature>http://tizen.org/feature/microphone</feature>
+        /// <param name="cmdType">Command Type</param>
+        /// <exception cref="InvalidOper
```

Comment:
Is it string or buffer?

Reply:
It is byte data.

Reply:
It is byte data. And I changed 'Marshal.PtrToStringAnsi' to 'Marshal.Copy'.